bundles/io/org.eclipse.smarthome.io.voice/src/main/java/org/eclipse/smarthome/io/voice/internal/text/StandardInterpreter.java

```diff
@@ -62,6 +73,10 @@ public void createRules() {
         itemRule(seq(cmd(alt("brighten", "increase", "harden", "enhance"),
             IncreaseDecreaseType.INCREASE), the) /* item */),
 
+        /* ColorType */
+
+        itemRule(seq(opt("set"), the, "color", of, the), /* item */ seq(to, color)),
```

Comment:
Right now the word "color" is required. What about another form of the rule that allows "set living room light to red"?

Or would it be okay to use opt("color") and allow "living room light red"?